### ISSUE TITLE
Provide a way to trigger auto-publishing docs for fresh forks

### ISSUE THREAD

**Issue by damithc** [se-edu/addressbook-level4#770]
As auto-publishing of docs is triggered by new commits to the master branch, docs of a fresh fork will not be published until there is a new commit. Is there a way to trigger the publishing even for fresh forks?

**Comment by pyokagan** [comment]
imo, while this is an issue, I'm not sure it is actually a big problem in practice. What's wrong with just adding a new commit?

**Comment by damithc** [comment]
Yes, not a big problem. Folks usually do the set up first before adding new commits. In this case students might end up adding dummy commits (to the master branch!) just to confirm that publishing is working.

**Comment by pyokagan** [comment]
@damithc 
> Yes, not a big problem. 

So, this can be `p.Low`?

> In this case students might end up adding dummy commits (to the master branch!) just to confirm that publishing is working.

Yes, that's bad if the ideal is to have a completely clean history, but regardless of whether we solve this issue or not the average student (or person) will still end up adding irrelevant commits to their master branch anyway. Considering that this setup is only done once at the beginning of the project, the impact is quite minimal.

**Comment by damithc** [comment]
> So, this can be p.Low?

Agree. Priority lowered.

> Yes, that's bad if the ideal is to have a completely clean history, but regardless of whether we solve this issue or not the average student (or person) will still end up adding irrelevant commits to their master branch anyway. Considering that this setup is only done once at the beginning of the project, the impact is quite minimal.

Perhaps the dev guide can mention how to create an empty commit?
